src/main/java/com/google/cloud/spanner/pgadapter/statements/BackendConnection.java

```diff
@@ -745,6 +784,32 @@ void execute() {
         Suppliers.memoize(
             () -> new PgCatalog(BackendConnection.this.sessionState, wellKnownClient.get()));
     this.spannerConnection = spannerConnection;
+    this.spannerConnection.addTransactionRetryListener(
+        new TransactionRetryListener() {
+          @Override
+          public void retryStarting(
+              Timestamp transactionStarted, long transactionId, int retryAttempt) {
+            logger.log(
+                Level.WARNING,
```

Comment:
should this be warning or info ? Considering retrials are expected in most scenarios

Reply:
Yeah, I had already reduced it to FINE.